.changeset/wicked-carrots-greet.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/avatar': patch
```

Comment:
Could this be a breaking change? If so, is it worth it? Can check reporting too

Reply:
Its not a breaking change, I just misread the code initially. 

The logic was happening in 2 places before: 

In the Avatar component, it checked if the src prop was there and rendered an image. If it wasn't there, it rendered an icon or initials

Then, in the AvatarContents component it did the opposite,  if the icon prop was present, it rendered that. Then if the src was there it would render an image, then the initials.

I caused the confusion because i didn't see that the src attribute was being checked in the Avatar component first. So, it was always image, icon, then initials.

To also clear this up in the future, I simplified the logic. You can see now that all the logic is in the AvatarContents component.